src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -18,9 +19,34 @@ import {
   FieldLabel,
   FieldSeparator,
 } from "../../../../styles/editMeasure/populationCriteria/supplementalData";
+import { MenuItem as MuiMenuItem } from "@mui/material";
+
+const improvementNotationOptions = [
+  {
+    label: "-",
+    value: "",
+    subtitle: null,
+    code: null,
+  },
+  {
+    label: "Increased score indicates improvement",
+    value: "Increased score indicates improvement",
+    subtitle:
+      "Improvement is indicated as an increase in the score or measurement (e.g. Higher score indicates better quality).",
+    code: "increase",
+  },
+  {
+    label: "Decreased score indicates improvement",
+    value: "Decreased score indicates improvement",
+    subtitle:
```

Comment:
subtitles going to be used for something in the future?

Reply:
I think so, I just copied what we have for Qi-Core. I thought about it, but since we are unsure why it was added in the first place, I left it as is.